### ISSUE TITLE
query: how to add filter to not show warning on matching some string?

### ISSUE THREAD

**Issue by p00j4** [PyCQA/pylint#1350]
```
pylint 1.6.4
python 2.7.11
```
### Steps to reproduce
I have "cell-var-from-loop" enabled which also shows warning for lambda, map functions
sample code breaks with "__init__.<lambda>): [W0640: "Cell variable r defined in loop"]"
```
def square(x):
            return (x**2)
        def cube(x):
            return (x**3)

        funcs = [square, cube]
        for r in range(5):
            value = map(lambda x: x(r), funcs)
            print value
``` 
How to specify use this type cell-var-from-loop but ignore for "lambda"

**Comment by rogalski** [comment]
For what I know, currently it's not possible.

**Comment by PCManticore** [comment]
This is currently not possible.

**Comment by p00j4** [comment]
oh, ok I get it but this way, it doesn't allow to have lambda, map etc functions and for CI. would love to this feature. I'm also trying read docs and understand if this is doable and I can send the PR
as work around, for now going with grep way

**Comment by rogalski** [comment]
The thing is - **it is** _cell-var-from-loop_ violation, even if it's not visible in code you're posted.

Common technique to avoid this behavior is to add extra argument with default value:
```
for r in range(5):
    value = map(lambda x, r=r: x(r), funcs)
    print value
```

This does not cause a violation and will be more bulletproof if anyone would introduce a modification to code snippet shown below.

**Comment by p00j4** [comment]
nice, thank you Łukasz 👍 
learned a good thing today.
